roles/prometheus/defaults/main.yml

```diff
@@ -23,3 +23,5 @@ conda_store_prefix: "/conda-store"
 
 # role: keycloak
 keycloak_port: "30020"
+
+prometheus_additional_scrape_configs: []
```

Comment:
Could this be added around line 10 above the "roles..." sections. Otherwise looks great!

Reply:
Yeah for sure! Updated